README.md

```diff
@@ -8,6 +8,10 @@ Terminus plugin to open any Pantheon site database using a SQL GUI client.
 
 ## Installation
 
+## Prerequisites
+Plugins directory must exist inside the terminus directory
+`mkdir ~/.terminus/plugins`
```

Comment:
It should be noted this is only necessary for terminus 2.x.  It might be easier to include it on the same line as `composer create-project ...` like below:
```
mkdir -p ~/.terminus/plugins && composer create-project --stability=beta -d ~/.terminus/plugins/ terminus-plugin-project/terminus-pancakes-plugin:~2
```
The install section can be updated for Terminus 3.x also.  See https://github.com/pantheon-systems/terminus-plugin-example#installation.

Reply:
@tekNorah: Please add this to the Terminus v2 section and I can get this merged.